### DISCUSSION TITLE
react

### DISCUSSION THREAD

**Discussion by mohammadsedigh** [community/community#123922] [Actions]
### Select Topic Area

Question

### Body

Select Topic Area
Question

Body
react

**Comment by MohamadSD2002** [comment]
response #123

**Reply by mohammadsedigh** [reply]
response #1233

**Comment by ebndev** [comment]
Hi @mohammadsedigh & @MohamadSD2002,

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
